lint-staged.config.js

```diff
@@ -1,14 +1,34 @@
 const escape = require('shell-quote').quote;
+const fs = require('fs');
 const isWin = process.platform === 'win32';
 
 module.exports = {
-  '**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}': filenames => {
+  '**/*{.css,.json,.md}': filenames => {
     const escapedFileNames = filenames
+      .filter(filename => fs.existsSync(filename))
       .map(filename => `"${isWin ? filename : escape([filename])}"`)
       .join(' ');
     return [
       `prettier --write ${escapedFileNames}`,
       `git add ${escapedFileNames}`
     ];
+  },
+  '**/*{.ts,.tsx}': filenames => {
+    const escapedFileNames = filenames
+      .filter(filename => fs.existsSync(filename))
+      .map(filename => `"${isWin ? filename : escape([filename])}"`)
+      .join(' ');
```

Comment:
Given this same "line" is used in all three cases it's probably better to make it a const function and call it in each instead.